docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   postgres:
-    image: postgres:11
```

Comment:
Didn't we have to do something special when we updated the ODB to a newer Postgres?